src/theme/material/avatar.m.css

```diff
@@ -0,0 +1,54 @@
+@import './variables.css';
+
+.root {
+	font-size: 1.25rem;
+	display: flex;
+	position: relative;
+	overflow: hidden;
+	align-items: center;
+	justify-content: center;
+	user-select: none;
+}
+
+.avatarColor {
+	color: var(--mdc-theme-surface);
+	background-color: var(--mdc-theme-primary);
+}
+
+.avatarColorSecondary {
+	color: var(--mdc-theme-surface);
+	background-color: var(--mdc-theme-secondary);
+}
+
+.small {
+	width: var(--mdc-avatar-size-small);
+	height: var(--mdc-avatar-size-small);
+}
+
+.medium {
+	width: var(--mdc-avatar-size-medium);
+	height: var(--mdc-avatar-size-medium);
+}
+
+.large {
+	width: var(--mdc-avatar-size-large);
+	height: var(--mdc-avatar-size-large);
+}
+
+.circle {
+	border-radius: 50%;
+}
+
+.square {
+	border-radius: 0;
+}
+
+.rounded {
+	border-radius: var(--mdc-border-radius);
+}
+
+.image {
+	object-fit: cover;
```

Comment:
I'm not sure that this works in ie11 or edge. How about using `background-image` instead then you would no longer need to create the img element

Reply:
Ooh, good call. I don't believe that's supported by IE11.